clients/cobol-lsp-vscode-extension/src/test/runTest.ts

```diff
@@ -21,7 +21,11 @@ import * as process from "process";
 async function main() {
   try {
     // prepare development and tests paths
-    const extensionDevelopmentPath = path.join(__dirname, "../../");
+    const extensionDevelopmentPath = [
```

Comment:
I was not aware that we could use an array for the `extensionDevelopmentPath`. When I tried to use it from command line I was not able to get it to work.

Reply:
Nice! Got it to work now:
```sh
code --extensionDevelopmentPath=$PWD/cobol-lsp-vscode-extension --extensionDevelopmentPath=$PWD/idms-dialect-support --extensionDevelopmentPath=$PWD/daco-dialect-support  path/to/project/with/source/code/containing/dialect/usage
```

Reply:
Thank you @ilidio-lopes.